### PR TITLE
Fix TC7 and add PCxx pins for SAMR.

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: samd/samd21/pins.c [hunk 1]
```diff
@@ -526,3 +526,28 @@ PIN(PB03, EXTINT_CHANNEL(3), ADC_INPUT(11), TOUCH(9),
     #endif
     NO_TIMER);
 #endif
+
+// For SAMR. They have one function so its hardcoded elsewhere.
```
Comment (dhalbert) [inline samd/samd21/pins.c]: Do you mean they talk to the radio?

**Review by dhalbert — CHANGES_REQUESTED** [review]
Looks good! Just explicate the comment if it makes sense.

**Comment by tannewt** [comment]
Ok. I elaborated in the comment.

**Review by dhalbert — APPROVED** [review]
Great! Thanks!

**Pull request merged**